catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/impl/SubscriptionsPersistentStoreImpl.java

```diff
@@ -44,7 +44,7 @@
 
   private static final String EMAIL_PROPERTY = "emails";
 
-  private static final String ID = "id";
+  private static final String ID = "'id'";
```

Comment:
❓ Instead of changing this constant, would changing `queryId()` to `return quote(ID) + "=" + quote(id);` be a better option? Would adding the `'` around `id` in the constant negatively affect https://github.com/bdeining/ddf/blob/9530fda36f34d6125e22db053cf3eb98bb4dbd0e/catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/impl/SubscriptionsPersistentStoreImpl.java#L212-L218?

Reply:
fixed